### PR TITLE
Include/Remove/Move Missing/Unnecessary/Misplaced Libraries

### PR DESCRIPTION
**Pull request by micttyl**
Minor Style Fix

### CODE REVIEW

File: toolchain/semantics/semantics_ir.cpp [hunk 1]
```diff
@@ -5,13 +5,19 @@
 #include "toolchain/semantics/semantics_ir.h"
 
 #include "common/check.h"
-#include "llvm/Support/FormatVariadic.h"
-#include "toolchain/lexer/tokenized_buffer.h"
+#include "toolchain/semantics/node_kind.h"
+#include "toolchain/semantics/node_ref.h"
+#include "toolchain/semantics/node_store.h"
```
Comment (jonmeow) [inline toolchain/semantics/semantics_ir.cpp:10]: There's going to be a close link between semantics_ir.cpp and semantics_ir.h: let's not duplicate #includes between them.

File: toolchain/semantics/semantics_ir.cpp [hunk 2]
```diff
@@ -5,13 +5,19 @@
...
+#include "toolchain/semantics/nodes/binary_operator.h"
+#include "toolchain/semantics/nodes/function.h"
+#include "toolchain/semantics/nodes/integer_literal.h"
+#include "toolchain/semantics/nodes/return.h"
+#include "toolchain/semantics/nodes/set_name.h"
```
Comment (jonmeow) [inline toolchain/semantics/semantics_ir.cpp:15]: I think node_store.h _must_ include all individual nodes; I wasn't duplicating #includes as a consequence.

Comment (micttyl) [reply toolchain/semantics/semantics_ir.cpp:15]: There is only a practice here. We can technically not putting any headers and get away with it by Bazel. Given what are included from other files, we may want to see where each Print comes. Also, I am aware of classic programmers who may use good old tools and I want them to freely index it.

**Review by jonmeow — CHANGES_REQUESTED** [review]
I don't see any edits, did you forget to push changes?

**Comment by jonmeow** [comment]
I'll note, I'm in the midst of making significant changes to the semantics directory. I'm happy to accept small edits, but it could also make sense to leave it alone for now.

**Comment by micttyl** [comment]
> I'll note, I'm in the midst of making significant changes to the semantics directory. I'm happy to accept small edits, but it could also make sense to leave it alone for now.

Thank you. Then, I will look around else where.

**Pull request closed without merging**